### PR TITLE
Add play.OlympusClan.net

### PR DESCRIPTION
**Pull request by Meedony**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/olympusclan/manifest.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "server_name": "olympusclan",
+  "nice_name": "OlympusClan",
+  "direct_ip": "play.olympusclan.net",
+  "server_wildcards": [
+    "%.olympusclan.net"
+  ],
+  "social": {
+    "web": "https://www.olympusclan.net",
+    "web_shop": "https://store.olympusclan.net",
```
Comment (DoJapHD) [inline minecraft_servers/olympusclan/manifest.json:10]: https://i.imgur.com/W36rzn8.png

File: minecraft_servers/olympusclan/manifest.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  "server_name": "olympusclan",
+  "nice_name": "OlympusClan",
+  "direct_ip": "play.olympusclan.net",
+  "server_wildcards": [
+    "%.olympusclan.net"
+  ],
+  "social": {
+    "web": "https://www.olympusclan.net",
+    "web_shop": "https://store.olympusclan.net",
+    "web_support": "https://www.olympusclan.net",
```
Comment (DoJapHD) [inline minecraft_servers/olympusclan/manifest.json:11]: Web support cannot be forum; link FAQ or contact form; if non-existent, remove

File: minecraft_servers/olympusclan/manifest.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{
+  "server_name": "olympusclan",
+  "nice_name": "OlympusClan",
+  "direct_ip": "play.olympusclan.net",
+  "server_wildcards": [
+    "%.olympusclan.net"
```
Comment (DoJapHD) [inline minecraft_servers/olympusclan/manifest.json:6]: Not a real wildcard
https://i.imgur.com/EI1VQCY.png

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Server is unavailable due to maintenance, is it even released yet?

**Comment by Meedony** [comment]
The server is not released yet, but we have almost finished our work. We
want to ensure that it looks clean, and with LabyMod's support, I'm
confident it will look awesome.

**Comment by Meedony** [comment]
I've updated the manifest.json as requested.

**Comment by DoJapHD** [comment]
Sadly, a requirement is that the server has to be released; reopen PR once server is released

**Pull request closed without merging**